### DISCUSSION TITLE
Bash colors on Windows

### DISCUSSION THREAD

**Discussion by CidTori** [com-lihaoyi/mill#1351] [Q&A]
When I user mill.bat in cmd on Windows, I get this kind of output:
```
?[34m[3/42] mill.scalalib.ZincWorkerModule.worker ?[39m
?[1A?[2K?[9999D?[34m[5/42] app.sources ?[39m
?[1A?[2K?[9999D?[34m[10/42] app.allSources ?[39m
?[1A?[2K?[9999D?[34m[11/42] app.allSourceFiles ?[39m
?[1A?[2K?[9999D?[34m[13/42] app.resources ?[39m
?[1A?[2K?[9999D?[34m[30/42] app.compile ?[39m
...
```
It looks like color escape sequences used in bash for example (like `\e[34m`)
How can I have colored output on Windows ?
(Or at least: How can I remove bash color escape sequences?)

**Comment by alexarchambault** [comment]
This should have been fixed by https://github.com/com-lihaoyi/mill/pull/1308, which is part of mill `0.9.7` and `0.9.8`.

**Reply by CidTori** [reply]
I'm using 0.9.8 😕

**Reply by CidTori** [reply]
Ah it works with cmd but not inside IntelliJ's terminal

**Reply by CidTori** [reply]
Rectification: it works in ConEmu, but not in CMD, PowerShell and IntelliJ's Terminal

**Reply by CidTori** [reply]
Had to do [this](https://superuser.com/a/1300251/922107) to make it work everywhere.
Maybe mill can enable it explicitly so the users don't have to change the default with regedit or to rely on the terminal emulator's settings?
